includes/Core/Authentication/Clients/Google_Site_Kit_Client.php

```diff
@@ -243,4 +244,23 @@ protected function fetchAuthToken( OAuth2 $auth, callable $http_handler = null )
 	protected function handleAuthTokenErrorResponse( $error, array $data ) {
 		throw new Google_OAuth_Exception( $error );
 	}
+
+	/**
+	 * Executes deferred HTTP requests.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param RequestInterface $request Request object to execute.
+	 * @param string           $expected_class Expected class to return.
+	 * @return object An object of the type of the expected class or Psr\Http\Message\ResponseInterface.
+	 */
+	public function execute( RequestInterface $request, $expected_class = null ) {
+		$request = $request->withHeader(
+			'X-Goog-Quota-User',
+			sprintf( '%s-%s', get_home_url(), get_current_user_id() )
```

Comment:
This is probably fine but just a thought – what do you think about formatting it in a more structured way? i.e. `{scheme}{id}@{url-without-scheme}` like basic auth? e.g. `https://1@example.com` rather than `https://example.com-1`.

Reply:
Yep, i like it. Updated.